entities/required.js

```diff
@@ -0,0 +1,12 @@
+// File: required.js
+// Description: Lists all of the required courses in the CS applied option.
+
+const Required = Object.freeze({
+  courses: ["HHS 231", "HHS 241", "COMM 111", "COMM 114", "CS 160", "CS 161",
+    "CS 162", "CS 165", "CS 261", "CS 271", "CS 290", "MTH 231", "CS 225",
+    "MTH 231", "CS 225", "MTH 251", "MTH 252", "WR 121", "WR 214", "WR 222",
+    "ST 314", "WR 327", "CS 325", "CS 340", "CS 344", "CS 352", "CS 361",
+    "CS 362", "CS 372", "CE 372", "ECE 372", "CS 381", "CS 391", "CS 444",
+    "CS 461", "CS 462", "CS 463"]
+});
+exports.Required = Required;
```

Comment:
Good one. I think this is as close as we can get to Rob's idea of making editing the required courses easier.